md/data-management.md

```diff
@@ -0,0 +1,34 @@
+# Data Management in the UI Designer
+
+Discover how to explore [Business Data Model](define-business-data-model.md) (BDM) objects in the UI Designer, and use it to create user interfaces and variables.
+
+A **Data Model** tab is available from the Palette of the Page editor:
+
+![data_model_panel](images/data_model_panel.png)
```

Comment:
This screenshot is missing in your PR...

Reply:
And call it with uid prefix can be better

Reply:
It was not a very good idea, but I uploaded it directly from github. Here it is:

Reply:
Yes not a very idea. It's better to upload a new image in referenced PR ...